requirements-dev.txt

```diff
@@ -2,16 +2,16 @@
 black==22.10.0
 coverage==6.5.0
 coveralls==3.3.1
+django-stubs==1.12.0
+djangorestframework-stubs==1.7.0
 factory-boy==3.2.1
 Faker==15.1.1
 flake8==5.0.4
 isort==5.10.1
-pre-commit==2.20.0
-pytest-django==4.5.2
-responses==0.22.0
 
 # mypy and PEP 561 stubs
 mypy==0.982
-django-stubs==1.12.0
-djangorestframework-stubs==1.7.0
```

Comment:
I split the python stubs for readability. Why are we changing this?

Reply:
precommit is sorting it lexicographically